topics/proteomics/tutorials/proteogenomics-dbcreation/tutorial.md

```diff
@@ -386,20 +392,48 @@ along with the UniProt and cRAP databases.
 >   - {% icon param-select %} *"How are sequences judged to be unique?"*: `Accession and Sequence`
 >   - {% icon param-text %} *"Accession Parsing Regular Expression"*: `^>([^ |]+).*$`
 >
-{: .hands_on}
```

Comment:
I think this was removed by accident? It breaks the box currently

Reply:
oops